configs/logging_config.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import os
+import sys
+
+import structlog
+
+
+def configure_logging():
+    log_level = logging.DEBUG if os.getenv("LOG_LEVEL") == "DEBUG" else logging.INFO
```

Comment:
We should add something in the README about this env var and what it does? Or should @rmccar add it as part of his README update?